gemrb/core/GUI/GameControl.cpp

```diff
@@ -2079,6 +2075,15 @@ void GameControl::InitFormation(const Point &clickPoint)
 	SetCursor(core->Cursors[IE_CURSOR_USE]);
 }
 
+bool GameControl::TryDefaultTalk(Point p) const
```

Comment:
Pass the point by const reference, since it won't be touched.

Reply:
good idea, i'll make it